### PR TITLE
Build the shapes demo in GitHub Actions

### PR DESCRIPTION
**Pull request by mitza-oci**

### CODE REVIEW

**Review by iguessthislldo — APPROVED** [review]

File: .github/workflows/ishapes.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    - name: checkout OpenDDS
+      uses: actions/checkout@v2
+      with:
+        path: OpenDDS
+        submodules: true
+    - name: checkout ACE_TAO
+      uses: actions/checkout@v2
+      with:
+        repository: DOCGroup/ACE_TAO
+        ref: ace6tao2
```
Comment (jwillemsen) [inline .github/workflows/ishapes.yml:28]: Wouldn't it be safer to use the `Latest_Micro` tag?

**Comment by ClaytonCalabrese** [comment]
You will run out of disk space if you try to build `--static --tests`

**Comment by mitza-oci** [comment]
> You will run out of disk space if you try to build `--static --tests`

With `-t:ishapes` it's only building the `ishapes` target and its dependencies (just like `make ishapes`).

**Review by ClaytonCalabrese — APPROVED** [review]

**Comment by mitza-oci** [comment]
Since the new workflow is not run as part of CI, here's a link to it running in my fork https://github.com/mitza-oci/OpenDDS/actions/runs/1607595508

**Pull request merged**